x-pack/plugins/apm/server/routes/services/annotations/index.test.ts

```diff
@@ -218,21 +217,17 @@ describe.skip('getServiceAnnotations', () => {
       .mockImplementation(
         () =>
           new Promise((resolve, reject) => {
-            setTimeout(() => {
```

Comment:
For this test, the `setTimeout` is irrelevant since it will return the stored annotations anyway. But it's important to keep throwing an error when `getDerivedServiceAnnotations` is called to ensure it won't break the server.